config/samples/rbac/role.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: spanner-autoscaller-secret-reader
+  namespace: test
+rules:
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get"]
+    resourceNames: ["gcp-sa-myproject-myspanner"]
```

Comment:
```suggestion
    resourceNames: ["spanner-autoscaler-gcp-sa"]
```

Reply:
I have kept this secret name different on purpose because `manager-role` ClusterRole already has access to secrets named `spanner-autoscaler-gcp-sa` (in any namespace, since it is ClusterRole).
So this custom Role and RoleBinding is required only if the user wants to use a secret name which is different from `spanner-autoscaler-gcp-sa`.

Reply:
I see I understood.
Thanks!